ui/admin/tests/acceptance/host-catalogs/list-test.js

```diff
@@ -68,13 +70,20 @@ module('Acceptance | host-catalogs | list', function (hooks) {
       type: TYPE_HOST_CATALOG_DYNAMIC,
       plugin: { name: TYPE_HOST_CATALOG_PLUGIN_AZURE },
     });
+    instances.gcpHostCatalog = this.server.create('host-catalog', {
```

Comment:
non-blocking:  It looks like `gcpHostCatalog` is not defined in the instances object above.